build.xml

```diff
@@ -1,35 +1,101 @@
-<project name="Documentation" default="all.doc" >
-   <property name="pdf.name" value="bean-validation-specification.pdf"/>
-   <property name="spec.styles.dir" value="${basedir}/styles"/>
-   <property name="document.basedir" value="${basedir}"/>
-
-
-   <import file="${document.basedir}/docbook-support/support.xml"/>
-
-   <target name="all.doc" depends="clean">
-      <antcall target="lang.all">
-         <param name="lang" value="en"/>
-         <param name="basedir" value="${document.basedir}"/>
-         <param name="styles.dir" value="${spec.styles.dir}"/>
-      </antcall>
-   </target>
-
-   <target name="clean">
-      <delete dir="${document.basedir}/build" />
-   </target>
-
-   <target name="createTckAuditFile">
```

Comment:
Why do you need them back? They won't work with the other changes. If you need them as development reference, they are in git.. I often worked with that deletion reverted in my work-in-progress branch as they are obviously useful to make progress.